src/screens/TokenDetail.js

```diff
@@ -152,14 +152,14 @@ class TokenDetail extends React.Component {
 
     if (!this.state.token) return null;
 
-    const isNFT = () => {
-      return this.state.token.meta && this.state.token.meta.nft;
+    const hasNFTMedia = () => {
+      return this.state.token.meta && this.state.token.meta.data.nft_media;
     }
 
     return (
       <div className="content-wrapper flex align-items-center">
         <TokenAlerts token={this.state.token} />
-        { isNFT() ? 
+        { hasNFTMedia() ?
```

Comment:
Why use a ternary when you can just conditionally render inside the method?

Maybe it's personal, but I tend to find ternary operators hard to read and avoidable

Reply:
I agree that in cases where ternary have blocks with multiple lines of html. In this case, we're just deciding what component to render. I think that move it to a function with an `if` and 2 `return` statements is less readable